### ISSUE TITLE
Handle posting of extradata when some types need to be posted before the main object, some after

### ISSUE THREAD

**Issue by branchedelac** [FOLIO-FSE/folio_migration_tools#451]
For example organizations, where you need to post

1.1 Interfaces
1.2 Credentials
2 --Contacts--
3 Organizations
4. Notes

On one way to do it would be to add an argument to the batch poster that allows you to select which record types to POST.

**Comment by branchedelac** [comment]
Striking Contacts off the list as FOLIO allows you to post Organizations with UUIDs of Contacts not in FOLIO. Unitl the contact is loaded, the UI displays this as a "Removed contact".

**Comment by branchedelac** [comment]
For organizations, this works

1. Post Organizations
2. Post extra-data file containing Contacts, Notes, Interfaces, and Credentials

The only caveat is that the interface credential MUST be posted AFTER the interface it links to. Otherwise you get this error: "Interface credential id mismatch"

This works in the current implementation, where the credential is added to the extradata file after its interface, and where the batch poster posts extradata objects one by one.

```
contacts	{"phoneNumbers": [{"phoneNumber": "1425775", "isPrimary": true}], "addresses": [{"addressLine1": "The reading room", "isPrimary": true}], "firstName": "Henry", "lastName": "Baskerville", "notes": "Enquire via email about dog for sale. It's very adorable.", "id": "299bf936-c683-495a-b7d5-8ee6c1f54f48"}
notes	{"typeId": "f5bba0d2-7732-4687-8311-a2cb0eaa12e5", "title": "Migrated note", "domain": "organizations", "content": "Watch out for dogs.", "links": [{"id": "bbf61aa3-05ea-5d15-99c8-e3e547001543", "type": "organization"}]}
interfaces	{"name": "British Beekeeper's Association", "uri": "https://www.bbka.org.uk/", "available": true, "id": "bcd79290-19d7-48a9-9448-82a2a910bbb7"}
interfaceCredential	{"username": "beekeeperno1", "password": "beesknees", "interfaceId": "bcd79290-19d7-48a9-9448-82a2a910bbb7", "id": "cd15264c-e941-4665-98bc-cfaf25e0fe8c"}
```

**Comment by branchedelac** [comment]
Closing as this does not seem to be a problem with organizations. We will create a new one if it we think there is any other area where it might be an issue.
